radixdlt-java/radixdlt-java/src/test/java/com/radixdlt/client/lib/api/sync/SyncRadixApiCreationTest.java

```diff
@@ -220,7 +220,16 @@ public void testCreateFixedSupplyToken() {
 			.onFailure(failure -> fail(failure.toString()))
 			.onSuccess(client -> client.transaction().build(request)
```

Comment:
The formatting of this whole method is hard to understand imho.

Reply:
I would do something like that 

```
		RadixApi.connect(BASE_URL)
			.map(RadixApi::withTrace)
			.onFailure(failure -> fail(failure.toString()))
			.onSuccess(client ->
				client.transaction().build(request)
				.onFailure(failure -> fail(failure.toString()))
				.onSuccess(builtTransaction -> {
					assertEquals(amount(100116600).micros(), builtTransaction.getFee());

					var notifications = builtTransaction.getTransaction().getNotifications();
					assertEquals(1, notifications.size());
					assertEquals("TokenCreate", notifications.get(0).getType());
					assertEquals("fix", notifications.get(0).getSymbol().orElse(""));
					assertEquals("fix_dr1qd5ah4xye2svl2smkk06st3q9lkqnkc6wgy0zxrrkmyszlawrz",
								 notifications.get(0).getRri().orElse(""));
				})
				.map(builtTransaction -> builtTransaction.toFinalized(KEY_PAIR1))
				.onSuccess(finalizedTransaction ->
					client.transaction().finalize(finalizedTransaction, true)
					.onFailure(failure -> fail(failure.toString()))
				)
			);
```
are we using a code formatter tool?

Reply:
Reformatted the whole class. Hopefully this will make it more readable.

Reply:
This is a matter of taste. Actually, initial formatting is more convenient and readable for me. Proposed one keeps parameter name on a separate line. This is a) inconsistent with a single line lambda formatting; b) requires to visually "lookup" parameter name, which is distracting.

Reply:
Agree, not that good either. I was trying to make it easier to know when a method starts and finishes.